app/Http/Requests/ImportRequests/ImportServerRequest.php

```diff
@@ -27,6 +27,9 @@ public function rules()
 			'path' => 'string|required',
 			'albumID' => 'int|required',
 			'delete_imported' => 'int',
+			'import_via_symlink' => 'int',
```

Comment:
you need to add `sometimes` or `nullable` for those next three. 
https://laravel.com/docs/8.x/validation#available-validation-rules

Reply:
Why? The front end always sets them, as far as I can tell. Were you testing with the old front end perhaps?

Reply:
Will double check.

Reply:
nevermind.